template.yml

```diff
@@ -21,34 +21,10 @@ Resources:
       ApplicationName: !Ref application
       EnvironmentName: BETA
       OptionSettings:
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: AWS_REGION
-          Value: !Ref AWS::Region
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: NOTIFICATION_TOPIC
-          Value: !Ref notificationTopic
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: NOTIFICATION_EMAIL
-          Value: !Ref emailAddress
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: USER_TABLE
-          Value: !Ref userTable
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: SESSION_TABLE
-          Value: !Ref sessionTable
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: GAME_TABLE
-          Value: !Ref gameTable
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: MOVE_TABLE
-          Value: !Ref moveTable
-        - Namespace: aws:elasticbeanstalk:application:environment
-          OptionName: STATE_TABLE
-          Value: !Ref stateTable
         - Namespace: aws:autoscaling:launchconfiguration
           OptionName: IamInstanceProfile
           Value: !Ref instanceProfile
-      PlatformArn: arn:aws:elasticbeanstalk:::platform/Java 8 running on 64bit Amazon Linux
+      PlatformArn: arn:aws:elasticbeanstalk:::platform/Corretto 8 running on 64bit Amazon Linux 2
```

Comment:
When I looked into Corretto, there were additional changes required, such as moving the proxy configuration to `.platform` from `.ebextensions`, and updating the Gradle version. I started this work in the [corretto](https://github.com/aws-samples/eb-java-scorekeep/compare/corretto) branch. Please take a look at that.

Reply:
I think you're right. Getting this sample app to run on Corretto 11 would likely need Gradle v5+ and other changes. When I tried running this code of Corretto 11, I immediately got build failures. 
But I discussed with @willarmiros and we are okay with supporting only Corretto 8 in the scope of this PR.

Reply:
Yep - upgrading gradle was too large of an effort. Let's stick with Java 8 corretto, and we will plan to document the incompatibility for now.

Reply:
Corretto 8 is fine. I didn't see a build error on v1.3 in Java 8 but did encounter a different error when using the random name option in the frontend (`com.fasterxml.jackson.core.JsonParseException: Unexpected character` in `scorekeep.UserFactory.randomName`).

Regarding the proxy configuration, Beanstalk does [require this to be in `.platform/nginx` instead of `.ebextensions/nginx`](https://docs.aws.amazon.com/elasticbeanstalk/latest/dg/using-features.migration-al.html) for AL2. Moving it there causes the deployment to fail. However, if it's removed or left in `.ebextensions` the application works. [nginx.conf](https://github.com/aws-samples/eb-java-scorekeep/blob/xray-gettingstarted/.ebextensions/nginx/nginx.conf#L25-L43) serves the `public` directory at root on port 80, and forwards traffic to `/api` to port 5000 where the Spring API is listening. Without it, Beanstalk sends all traffic to port 5000 and Spring serves the contents of `public` at the root path. So I think we can remove the proxy configuration. Can you confirm?

Reply:
With my limited knowledge of traffic routing within Beanstalk using this nginx config, I can't confirm this. 
Wouldn't it be best to test the app with the proxy removed? Is it something that must be done since the app seems to work fine?